qcelemental/tests/test_molparse_pubchem.py

```diff
@@ -38,12 +38,12 @@
         0.000000, -2.139300, 1.256400, 0.000100, -0.018400, -2.480900, -0.000100, 0.018400, 2.480800, 0.000000,
         2.139400, -1.256300, 0.000100, 2.157700, 1.224500, 0.000000
     ]),
-    'elbl': np.array(['C', 'C', 'C', 'C', 'C', 'C', 'H', 'H', 'H', 'H', 'H', 'H']),
     'elea': np.array([12, 12, 12, 12, 12, 12, 1, 1, 1, 1, 1, 1]),
     'elez': np.array([6, 6, 6, 6, 6, 6, 1, 1, 1, 1, 1, 1]),
     'elem': np.array(['C', 'C', 'C', 'C', 'C', 'C', 'H', 'H', 'H', 'H', 'H', 'H']),
     'mass': np.array([12., 12., 12., 12., 12., 12., 1.00782503, 1.00782503, 1.00782503, 1.00782503, 1.00782503, 1.00782503]),
     'real': np.array([True, True, True, True, True, True, True, True, True, True, True, True]),
+    # 'elbl': np.array(['C', 'C', 'C', 'C', 'C', 'C', 'H', 'H', 'H', 'H', 'H', 'H']),
```

Comment:
there's two definitions of `elbl` -- sort of a pre-parse and post-parse/storage. this is probably a residual from what I needed for testing earlier. can remove the "filled" line.